public-api/src/main/java/org/pmiops/workbench/cdr/dao/AchillesAnalysisDao.java

```diff
@@ -18,6 +18,16 @@
     )
     AchillesAnalysis findConceptAnalysisResults(String concept_id,Long analysisId);
 
+    @Query(value = "select a from AchillesAnalysis a where a.analysisId=?1")
+    AchillesAnalysis findAnalysisById(Long analysisId);
+
+    @Query(value = "select distinct a from AchillesAnalysis a left join FETCH a.results as r " +
+            "where r.stratum1 = ?1 and a.analysisId in (3101,3102,3103,3104) order by a.analysisId"
```

Comment:
Should we make the analysis_id a list the controller can pass in ? Then this function can be findConceptAnalysesResults --- Then it is generally useful... I like that.